index.html

```diff
@@ -1,47 +1,81 @@
-           <li>|</li>
-           <li><a class="links" href="projects.html">PROJECTS</a></li>
-       </ul>
-       </nav>
-      
-<main>
-        <div class="box1">
+      <img
+        class="image"
+        src="./assets/Dhruv.jpeg.jpg"
+        alt="logo"
+        width="50px"
+        height="50px"
+        style="border-radius: 100%"
+      />
+      <ul class="menu">
+        <li><a class="links" href="index.html">HOME</a></li>
+        <li>|</li>
+        <li><a class="links" href="projects.html">PROJECTS</a></li>
+      </ul>
+    </nav>
+
+    <main>
+      <div class="box1">
         <h1>Hey There!</h1>
+        <p>
+          Welcome to my portfolio. My name is Dhruv gupta from Gwalior, India. I
+          am a B.Tech graduate from mechanical engineering who is pursuing to
+          become an allrounder in the field of website develoment. I gained
+          interest in this field recently while making my bolg on wordpress, but
+          always thought i could do more if given the chance. I am a fitness
+          enthusiast on the side and would like to make my own website one day
+          through which I could earn my living.
+        </p>
+      </div>
+      <div class="box3">
+        <img
+          class="image"
+          src="./assets/Dhruv.jpeg.jpg"
+          alt="logo"
+          width="220px"
+          height="220px"
+          style="border-radius: 10%"
+        />
+        <div class="centerdiv">
+          <a class="links" href="https://github.com/Thesoulfit"
+            ><i class="fab fa-github"></i
+          ></a>
+          <a
+            class="links"
+            href="https://www.linkedin.com/in/dhruv-gupta-88425a141/"
+            ><i class="fab fa-linkedin"></i
+          ></a>
+          <a class="links" href="mailto:gupta.dhruv2695@gmail.com"
+            ><i class="fas fa-envelope-square"></i
+          ></a>
         </div>
-        <div class="box2">
-        <p> Welcome to my portfolio. My name is Dhruv gupta from Gwalior, India. I am a B.Tech graduate from mechanical engineering who is pursuing to become an allrounder in the field of website develoment.
-             I gained interest in this field recently while making my bolg on wordpress, but always thought i could do more if given the chance. I am a fitness enthusiast on the side and would like to make my own website one day through which I could earn my living. 
-            </p>
-        </div>
-        <div class="box3">
-            <img class = "image" src="./assets/Dhruv.jpeg.jpg" alt="logo" width="220px" height="220px" style="border-radius: 10%;">
-            <div class="centerdiv">
-                <a class="links" href="https://github.com/Thesoulfit"><i class="fab fa-github"></a></i>
-                <a class="links" href="https://www.linkedin.com/in/dhruv-gupta-88425a141/"><i class="fab fa-linkedin"></a></i>
-                <a class="links" href="mailto:gupta.dhruv2695@gmail.com"><i class="fas fa-envelope-square"></a></i>

```

Comment:
Your html tags were in an incorrect order.
You were closing the `<a></a>` tag before de`<i></i>` 
you have the correct syntax on the right, but is basically this one:
```html
     <a class="links" href="mailto:gupta.dhruv2695@gmail.com">
          <i class="fas fa-envelope-square"></i>
     </a>
```

Try to use some format extension in vsCode that can help you spot this errors